.vscode/launch.json

```diff
@@ -5,11 +5,11 @@
       "name": "C/C++ Runner: Debug Session",
       "type": "cppdbg",
       "request": "launch",
-      "args": [],
       "stopAtEntry": false,
       "externalConsole": false,
-      "cwd": "/home/jpn323/workspace/rdk-halif-hdmi_cec/ut/ut-core/include",
-      "program": "/home/jpn323/workspace/rdk-halif-hdmi_cec/ut/ut-core/include/build/Debug/outDebug",
+      "cwd": "/home/jpn323/workspace/rdk-halif-hdmi_cec/ut/ut-core/tests/bin/",
```

Comment:
you need to rebase this branch now, but it's got hardcoded values here they should dissappear.